src/mongo/MongoTripodConfig.class.php

```diff
@@ -296,6 +318,14 @@ protected function loadConfig(Array $config)
                 {
                     throw new MongoTripodConfigException("View spec does not contain " . _ID_KEY);
                 }
+                if(!isset($spec['from']) || !in_array($spec['from'], $this->getPods($storeName)))
```

Comment:
Originally, I had done this, but, actually, the amount of validation done for search and view specs is the same as we do inline for table specs.

Really, it's just that we do *a lot* more validation for table specs.